README.md

```diff
@@ -260,6 +260,39 @@ npm run code
 npm test
 ```
 
+## Plugins
+
+Sinon chrome module supports by plugins.
+
+### Cookie plugin
+
+You can test chrome cookie behaviour.
+
+#### install
+
+```js
+var chrome = require('sinon-chrome');
+var cookiePlugin = require('sinon-chrome/out/plugins/cookies');
```

Comment:
Agee with @andre487 !
1. move plugins to `src/plugins`
2. rename `install` to `installPlugin` or `attachPlugin` to be more specific
3. i suggest to prefix all additional non-standart chrome.api to exclude confusion for developers (someone may want to search for `chrome.install` in chrome docs). I vote for `chrome._attachePlugin()` and `chrome.cookies._state = ...`